# Loosely-Coupled, Error (15) State EKF Message Structure


Header header

State state

Covariance covariance

Unbiased_Inputs inputs